ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    public class CreateModel : EventSpecificPageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
+
+        public CreateModel(ServerCore.Models.PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        public IActionResult OnGet()
+        {
+            return Page();
+        }
+
+        [BindProperty]
+        public Team Team { get; set; }
+
+        public async Task<IActionResult> OnPostAsync()
+        {
+            if (!ModelState.IsValid)
+            {
+                return Page();
+            }
+
+            Team.Event = Event;
+            Team.PuzzleCacheLastUpdated = DateTime.UtcNow;
```

Comment:
Puzzle cache is a somewhat suspect holdover from the old system. I'd suggest ditching it.

Reply:
If @asyasky has another plan here I am all for it.

Reply:
What exactly is it caching? (puzzle list, puzzle state, something else?)

Reply:
I think it was caching something puzzle list and state related

Reply:
Sounds like there are no plans for this. I will remove when the rest of the issues are resolved.